wp-content/themes/midwestenergynews/partials/sidebar-digest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/*
+ * For the digest category page
+ *
+ * Copied from Largo's partials/sidebar-archive.php at 0.6.4
+ * @package Largo
+ * @since 0.6.4
+ * @link https://github.com/INN/umbrella-energynewsnetwork/issues/86
+ */
+$custom_sidebar = largo_get_custom_sidebar();
+if ( !dynamic_sidebar( 'digest-sidebar' ) ) { // try custom sidebar registered in functions.php
+	// do nothing.
+	echo 'aaa';
```

Comment:
Probably want to remove this 😂

Reply:
Fixed in https://github.com/INN/umbrella-energynewsnetwork/pull/89/commits/57ec81573ab19db78bfa3b15d716018191ecfbe1 🤦‍♂